### PR TITLE
Create ZAC.md

### PR DESCRIPTION
**Pull request by balhar-jakub**
Move the key readme.md from zac repository to this repository.

Add issue template for zac requests

### CODE REVIEW

**Comment by armstro** [comment]
image
Thanks Jakub - I assume "template" means predefined fields (like taking the issue with zac label) and will show on the new issue page? Are there any questions we should ask in the template to confirm it is an issue for the zac? I will post this to the zac slack channel for awareness of others.

File: ZAC.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ - Create and maintain, in partnerhsip with the On-Boarding squad, a master list of events where Zowe should participate and corrdinate getting the Zowe message out 
+ - The ZAC has a goal of consistent Zowe project messaging in presentations, blogs, images, words and actions 
+ 
+The ZAC will take on other responsiblities as agreed with the Zowe TSC. The ZAC will take the lead from the Zowe community with the Open Mainframe Project when the Zowe TSC agrees. 
+
+## Membership
+
+The ZAC consists of up to 9 voting members. 
+
+One vote is assigned to the TSC. The TSC decides who their voting member is. The remaining 8 ZAC members will come from Open Mainframe Project Silver (or above) members. Members are nominated by any Zowe committer. No more than 2 individuals on the ZLC can be affiliated with the same company. The nominee must agree to be a candidate and to serve if elected. ZAC members should be willing to actively serve by attending weekly ZAC meetings and voting on ZAC issues for at least a one year term. ZAC members are voted on by the Zowe committers.
```
Comment (pinpan) [inline ZAC.md:25]:
```suggestion
One vote is assigned to the TSC. The TSC decides who their voting member is. The remaining 8 ZAC members will come from Open Mainframe Project Silver (or above) members. Members are nominated by any Zowe committer. No more than 2 individuals on the ZAC can be affiliated with the same company. The nominee must agree to be a candidate and to serve if elected. ZAC members should be willing to actively serve by attending weekly ZAC meetings and voting on ZAC issues for at least a one year term. ZAC members are voted on by the Zowe committers.    
```

File: ZAC.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Zowe Advocacy Council (ZAC)
+
+## Roles and Responsibilities
+
+The Zowe Advocacy Council (ZAC) is a two way communication and coordination team primarily, but not limited to, commercial exploiters of Zowe. This two way channel is for project requirements from commercial stake-holders and to assist in outbound messaging of Zowe to the z/OS ecosystem. The ZAC also manages new technology directions for the Zowe project via approving new sub-projects (i.e., incubator projects) and is an arbitor of any major issues needing resolution that impact the whole of the Zowe project.
```
Comment (pinpan) [inline ZAC.md:5]:
```suggestion
The Zowe Advisory Council  (ZAC) is a two way communication and coordination team primarily, but not limited to, commercial exploiters of Zowe. This two way channel is for project requirements from commercial stake-holders and to assist in outbound messaging of Zowe to the z/OS ecosystem. The ZAC also manages new technology directions for the Zowe project via approving new sub-projects (i.e., incubator projects) and is an arbitor of any major issues needing resolution that impact the whole of the Zowe project.  
```

File: ZAC.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+The 8 seats are voted on yearly (typically Feburary) with the top 8 vote getters to serve. In the event there are not 8 nominees (or a nominee receices no votes) then the members with votes shall fill the seats and empty seats will exist. 
+
+Empty seats can be filled at any time during the year. New OMP Silver (or above) members can be nominated by any existing ZAC members. ZAC members vote to fill an empty seat for the remainder of the 1 year term until the next February at-large vote on the ZAC members. Adding a new ZAC member to fill a vacant seat needs unanimous  agreement of the ZAC members. 
+
+The ZAC will operate on a consensus basis in all decisions.  The roles on the ZLC (Chair, Vice or Secretary) are decided by the members of the ZLC.
```
Comment (pinpan) [inline ZAC.md:31]:
```suggestion
The ZAC will operate on a consensus basis in all decisions.  The roles on the ZAC (Chair, Vice or Secretary) are decided by the members of the ZAC. 
```

**Pull request closed without merging**